v0/src/simulator/src/minimap.js

```
@@ -83,111 +82,97 @@
         this.ctx.fill()
 
-        //  to show the area of current canvas
-        var lst = updateOrder
         const miniFill = colors['mini_fill']
         const miniStroke = colors['mini_stroke']
 
         this.ctx.strokeStyle = miniStroke
         this.ctx.fillStyle = miniFill
-        for (var i = 0; i < lst.length; i++) {
+
+        const lst = updateOrder
```

ESLint

[error] 'updateOrder' is not defined.

(no-undef)